example.py

```
@@ -908,6 +910,15 @@ def switch(api, i):
                     "api.get_userprofile_settings()", api.get_userprofile_settings()
                 )
 
+            elif i == "X":
+                # Get latest lactate threshold
+                display_json(
+                    "api.get_lactate_threshold(latest=True)", api.get_lactate_threshold(latest=True)
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)